src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller.Session_FilterModel.cs

```diff
@@ -221,12 +221,20 @@ internal partial class Session
 
             private Boolean IsAfterDot(Model model, ITextSnapshot textSnapshot, Dictionary<TextSpan, string> textSpanToText)
             {
-                var span = model.OriginalList.Span;
```

Comment:
@CyrusNajmabadi tested out empty file/empty watch/empty immediate window and didn't find any issues.